tests/v2/test_waku_store.nim

```diff
@@ -595,7 +596,8 @@ procSuite "Waku Store":
     listenSwitch.mount(proto)
 
     for wakuMsg in msgList:
-      await subscriptions.notify("foo", wakuMsg)
+      # the pubsub topic should be DefaultTopic
```

Comment:
leftover comment?

Reply:
It is indeed intentional, the node should query for messages published on DefaultTopic, and it should not change in the future updates of this unit test.